src/stories/components/cutable-column-links/cutable-column-links.tsx

```diff
@@ -67,10 +68,14 @@ const Container = styled.div({
   alignItems: 'center'
 });
 
+type StickyLinkProps = {
+  width?: string | number,
+  height?: string | number,
```

Comment:
Is there a good reason why we are parsing both strings and numbers here?

Reply:
i will let only number